### ISSUE TITLE
Static analysis tools in CI chain

### ISSUE THREAD

**Issue by CptF1nn** [ITUDevUps/devUps#66]

**Comment by JensFastrup** [comment]
isn't this covered by SonarCube already?

**Comment by CptF1nn** [comment]
@JensFastrup No, static analysis tools are something that checks if things af compileable for example (see task 1 here https://github.com/itu-devops/lecture_notes/blob/master/sessions/session_07/README_TASKS.md)

**Comment by JensFastrup** [comment]
Static analysis tools, that check 
- Code quality
- Maintainability
- Security 
- Technical debt
- etc.
has been integrated with tools in the CI with Sonarcloud, Code Climate, Snyk
